src/main.js

```diff
@@ -115,6 +115,13 @@ function renderResults(resultsList) {
                         ? `&nbsp;&nbsp;&nbsp;<div style='background-color:#49adf4;border-radius:50%;padding:0.2rem 0.3rem;font-size:0.35em;height:fit-content;'>✔</div>`
                         : ''
                 }\
+                ${
+                    user.is_private
+                        ? `<div style="display:flex;width:100%;justify-content:space-around;">\
+                            <span style="border: 2px solid #51bb42;border-radius:25px;padding:0.5rem;color:#51bb42;font-weight:500;">Private</span>\
+                          </div>`
+                        : ''
```

Comment:
Doesn't look the best IMO.
Can be improved over time..

Reply:
> Maybe the private label should be aligned, but that's ok.

I agree, but need to see how to do it elegantly.
So this is something that could be improved over time.

> We should re-implement the possibility to see the unfollowers before finishing the whole cycle.

We could do this but it would raise some questions.

1. How do we indicate the scan status/percentage to the user if we start showing the results on screen?
2.  What options do we allow while scanning?
@davidarroyo1234